libgadget/utils/event.c

```diff
@@ -51,7 +51,7 @@ event_emit(EventSpec * eh, EIBase * event)
     return 0;
 }
 
-EventSpec EventSlotsFork = {"SlotsFork", 0};
-EventSpec EventSlotsAfterGC = {"SlotsAfterGC", 0};
+EventSpec EventSlotsFork = {"SlotsFork", 0, {{NULL}, {NULL}}};
```

Comment:
Is this necessary? I thought C compiler is supposed to zero pad the struct when the initializer is incomplete.

Reply:
{"SlotsFork", 0, {{0}}} works. Would you prefer that?

Reply:
The issue I'd like to avoid is when the struct grows, the initializer here shall not start to give warnings that we have to fix down the road. So `{{0}}`, as ugly as it is it solves this.